grinder/core.py

```diff
@@ -943,6 +1009,35 @@ def __censys_save_to_database(self, query: dict) -> None:
             query=query, results_count=results_count, results=results_by_query
         )
 
+    @exception_handler(expected_exception=GrinderCoreMasscanSaveToDatabaseError)
+    def __masscan_save_to_database(self, query: dict) -> None:
+        """
+        Save current query-based results to database
+
+        :param query: current search query
+        :return: None
+        """
+        def check_ip(address: str, network: str) -> bool:
```

Comment:
<img width="1244" alt="bug" src="https://user-images.githubusercontent.com/24269213/85234261-46f64600-b436-11ea-925f-5c3d03103349.png">

```
        "masscan_settings": [
            {
                "hosts": "92.63.64.1/28",
                "ports": "",
                "rate": "1000",
                "top-ports": "10"
            },
            {
                "hosts": "92.63.64.17/28",
                "ports": "",
                "rate": "1000",
                "top-ports": "10"
            },
            {
                "hosts": "92.63.64.33/28",
                "ports": "",
                "rate": "1000",
                "top-ports": "10"
            }
        ],
```
1. What's wrong?
2. Why the network was changed during Grinder processing? 

Take a look at the screenshot.

Reply:
> What's wrong?

Found ports are not included in the top-ports[:10] list:
MASSCAN_TOP_PORTS[:10] = ['1', '3', '4', '6', '7', '9', '13', '17', '19', '20']

If we set top-ports to 12:
MASSCAN_TOP_PORTS[:12] = ['1', '3', '4', '6', '7', '9', '13', '17', '19', '20', '21', '22']

And we will get the result:
![img](https://i.ibb.co/nLYnV40/Screenshot-from-2020-06-22-18-49-28.png)

> Why the network was changed during Grinder processing?

Module print real network address without host bits